src/app/SharpRaven/Data/BreadcrumbsLevel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SharpRaven.Data
+{
+    /// <summary>
+    /// This defines the level of the event. If not provided it defaults to info which is the middle level. In the order of priority from highest to lowest the levels are critical, error, warning, info and debug. Levels are used in the UI to emphasize and deemphasize the crumb.
+    /// </summary>
+    public enum BreadcrumbsLevel
+    {
+        Critical, Error, Warning, Info, Debug
+    }
+}
```

Comment:
Yes, it is. There are a bit of difference. The ErroLevel is really for error messages already breadcrumbLevel is about the level of the event track. And then I had to preferred retain both on specific contexts. The enums of both are different too.